src/features/rewards/siteBanner/style.ts

```diff
@@ -60,13 +60,20 @@ export const StyledBannerImage = styled<Partial<Props>, 'div'>('div')`
   font-weight: 600;
   line-height: 0.74;
   color: #d1d1db;
-  height: 176px;
+  min-height: 176px;
+  max-height: 350px;
```

Comment:
@cg505 It's not clear where this number 350px comes from? Might be uncomfortable to change in the future if we don't know why it's set at that number.

Reply:
this got removed with the change to brave-core